src/main/java/frc/team2412/robot/commands/launcher/SetPivotCommand.java

```diff
@@ -0,0 +1,26 @@
+package frc.team2412.robot.commands.launcher;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.team2412.robot.subsystems.LauncherSubsystem;
+
+public class SetPivotCommand extends Command {
+	private final LauncherSubsystem launcherSubsystem;
+	private final double angle;
+
+	public SetPivotCommand(LauncherSubsystem launcherSubsystem, double angle) {
+		this.launcherSubsystem = launcherSubsystem;
+		this.angle = angle;
+		addRequirements(launcherSubsystem);
+	}
+
+	@Override
+	public void initialize() {
+		launcherSubsystem.setAngle(angle);
```

Comment:
This command doesn't change the manual setpoint, so when it ends the pivot will go to the manual setpoint instead of holding the given position.

It may be better to have only one setpoint variable instead of having two separate variables for the manual and automated angle setpoints to avoid bugs like this.